dashboards-observability/public/components/visualizations/charts/dashboard_list/dashboard_list.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiBasicTable, EuiLink } from '@elastic/eui';
+
+export const DashboardList = ({ visualizations }: any) => {
+  const getTableData = () => {
+    const dashboards = [
+      {
```

Comment:
we need to showcase this and confirm on its implementation. Please create a video and discuss this in Monday call too.

Reply:
@abasatwar Video already created and updated ticket https://github.com/opensearch-project/observability/issues/970